### PR TITLE
Add privacy considerations around private IDs in glue URIs

### PR DESCRIPTION
**Pull request by brentzundel**
This PR introduces a privacy considerations section, and adds a subsection for considerations when private identifiers are used in corporate identifiers.

### CODE REVIEW

**Comment by brentzundel** [comment]
fixes #2

File: draft-zundel-spice-glue-id.md [hunk 1]
```diff
@@ -109,6 +109,19 @@ TODO DIDs and glue
...
+
+There are some corporate identifers which make use of personal identifiers. This
+is the case for registered sole-proprietor businesses in much of the United
+States, where the business identifier may be the same as the
+social-security-number of the business owner.
+
+It is possible for such identifers to be represented as glue URIs. An
+identifier's expression as a glue URI does not change the privacy
+characteristics of that identifier. The same cautions and concerns need to be
+taken with the glue URI representation as with the original identifier.
```
Comment (pamelatech) [inline draft-zundel-spice-glue-id.md:125]:
```suggestion
Implementers storing or evaluating glue ids are encouraged to evaluate the privacy characteristics of each authority potentially represented by authority identifiers and to appropriately handle any glue id whose authority identifier format violates privacy policies.
```

**Review by pamelatech — APPROVED** [review]
I added a suggestion of text that speaks directly to implementers, please take a look. Otherwise I think this clarifies the issue raised.

File: draft-zundel-spice-glue-id.md [hunk 2]
```diff
@@ -109,7 +109,20 @@ TODO DIDs and glue
...
+Implementers storing or evaluating glue ids are encouraged to evaluate the privacy characteristics of each authority potentially represented by authority identifiers and to appropriately handle any glue id whose authority identifier format violates privacy policies.
```
Comment (brentzundel) [inline draft-zundel-spice-glue-id.md:125]:
```suggestion
Implementers storing or evaluating glue ids are encouraged to evaluate the
privacy characteristics of each identification scheme represented by an authority
identifier and to appropriately handle any glue id which violates privacy policies.
```

**Pull request merged**